test/libsolidity/syntaxTests/inheritance/duplicated_super_constructor_call_multi.sol

```diff
@@ -0,0 +1,7 @@
+contract C { constructor(uint) public {} }
+contract A is C(2) {}
+contract B is C(2) {}
+contract D is A, B { constructor() C(3) public {} }
```

Comment:
Can you add another test where `D` does not have a constructor?

Reply:
And then perhaps also something like this:
```
+contract C { constructor(uint) public {} }
+contract A is C { constructor() C(2) {} }
+contract B is C { constructor() C(2) {} }
+contract D is A, B { }
```

Reply:
Ok - these cases are not caught yet and will complicate the detection - I will have a look into it.